nodejs/README.md

```diff
@@ -18,12 +18,15 @@ $ npm install
 ```
 
 Next you'll need to be sure to execute `node` with experimental flags. It's
-recommended to use Node.js 12+ here as well
+requires Node.js 13+ here as well
 
 ```
-$ node --experimental-modules --loader ./loader.mjs ./run.mjs
+$ node --experimental-wasm-mv --experimental-modules --loader ./loader.mjs ./run.mjs
 ```
 
+Note that Node.js 13 is required for multi-value support, the
+`--experimental-wasm-mv` flag.
```

Comment:
This sentence seems a bit off. Perhaps "Note that Node.js 13 with the `--experimental-wasm-mv` flag is required to enable the multi-value support needed by the polyfill`"?